serviceregistry-server/src/main/java/no/difi/meldingsutveksling/serviceregistry/model/BrregPostadresse.java

```diff
@@ -0,0 +1,40 @@
+package no.difi.meldingsutveksling.serviceregistry.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.Data;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+@Data
+@JsonIgnoreProperties
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Entity
+public class BrregPostadresse {
+
+    @Id
+    @GeneratedValue
+    private Integer pId;
+    private String adresse;
+    private String postnummer;
+    private String poststed;
+    private String land;
+
+    @JsonIgnore
+    public Integer getpId() {
+        return this.pId;
+    }
+
+    BrregPostadresse() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this constructor to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1213)